pkg/cache/v3/cache.go

```diff
@@ -122,6 +122,9 @@ type RawResponse struct {
 	// Resources to be included in the response.
 	Resources []types.ResourceWithTTL
 
+	// NextVersionMap consists of updated version mappings after this response is applied
```

Comment:
Comment is a bit unclear to me, as it doesn't tell what exactly is in the key and value. Suggestion:

```suggestion
	// NextVersionMap maps the resource name to the empty string for resources that were updated?
```

Reply:
Updated